war/src/main/js/pluginSetupWizardGui.js

```diff
@@ -613,6 +616,44 @@ var createPluginSetupWizard = function() {
 			$c.slideDown();
 		}
 	};
+	
+	// call to change to setup security
+	var setupSecurity = function() {
+		setPanel(securityPanel, {
+			baseUrl: jenkins.baseUrl
+		}, function() {
+		    console.log('frame has been added to the page');
+			$('iframe').load(function() {
+				var location = $(this).contents().get(0).location.href;
+			    console.log('frame has (re)loaded, is showing: ' + location);
+			});
```

Comment:
Also ... I'd put an id/class on that `iframe` so as to make sure you are selecting the right one. I'm sure there's just one there atm, but I wouldn't assume there's never going to be another (no need to assume it).